conda/shell/etc/profile.d/conda.csh

```diff
@@ -49,14 +49,14 @@ else
             $_CONDA_EXE $argv[1-]
             set ask_conda="`(setenv prompt '${prompt}' ; '${_CONDA_EXE}' shell.csh reactivate)`"
             set conda_tmp_status=$status
-            setenv PATH "$conda_tmp_path"
+            # setenv PATH "$conda_tmp_path"
             if( $conda_tmp_status != 0 ) exit ${conda_tmp_status}
             eval "${ask_conda}"
             rehash
             breaksw
         default:
             $_CONDA_EXE $argv[1-]
-            setenv PATH "$conda_tmp_path"
+            # setenv PATH "$conda_tmp_path"
```

Comment:
@kenodegard Didn't you want to clean these things up before merge?

Reply:
https://github.com/conda/conda/pull/11722